etl/steps/data/garden/owid/latest/population_density/__init__.py

```diff
@@ -53,21 +53,50 @@ def make_table() -> Table:
         .sort_values(["country", "year"])
     )
 
-    df.metadata.short_name = "population-density"
-    df.metadata.description = "Population density (World Bank, Gapminder, HYDE & UN)"
     assert (df.population_density >= 0).all()
     assert (df.population_density < 40000).all()
     return df.set_index(["country", "year"])
 
 
 def run(dest_dir: str) -> None:
     ds = Dataset.create_empty(dest_dir)
+    t = make_table()
+
+    source = Source(
+        name="World Bank (2021), Gapminder (v6), HYDE (v3.2), UN (2019)",
+        description='Our World in Data builds and maintains a long-run dataset on population by country, region, and for the world, based on three key sources: HYDE, Gapminder, and the UN World Population Prospects. This combines historical population estimates with median scenario projections to 2100. You can find more information on these sources and how our time series is constructed on this page: <a href="https://ourworldindata.org/population-sources">What sources do we rely on for population estimates?</a>\n\nWe combine this population dataset with the <a href="https://ourworldindata.org/grapher/land-area-km">land area estimates published by the World Bank</a>, to produce a long-run dataset of population density.\n\nIn all sources that we rely on, population estimates and land area estimates are based on today’s geographical borders.',
+        url="https://www.gapminder.org/data/documentation/gd003/; https://population.un.org/wpp/Download/Standard/Population/; https://dataportaal.pbl.nl/downloads/HYDE/; https://datacatalog.worldbank.org/dataset/world-development-indicators",
```

Comment:
Likewise here, if it was coming from the ETL we'd want these data sources separately.